sound/soc/sof/intel/hda-loader.c

```diff
@@ -290,8 +292,8 @@ static int cl_copy_fw(struct snd_sof_dev *sdev, struct hdac_ext_stream *hext_str
 
 	if (status < 0) {
 		dev_err(sdev->dev,
-			"error: %s: timeout HDA_DSP_SRAM_REG_ROM_STATUS read\n",
-			__func__);
+			"%s: timeout %#x read\n",
```

Comment:
@plbossart, I would consider @bardliao's suggestion to have more grepable print:
`"%s: timeout with rom_status_reg (%#x) read\n"` or even `"%s: timeout with ROM_STATUS (register: %#x) read\n"`

Reply:
ah yes, I removed rom_status_reg by mistake, the suggestion was the intended result, thanks for spotting this.

Reply:
done